src/aui-form-builder/js/aui-form-builder-layout-builder.js

```diff
@@ -455,6 +496,7 @@ A.FormBuilderLayoutBuilder.prototype = {
         if (!(targetNode.hasClass(CSS_MOVE_TARGET) && targetNode.hasClass(CSS_MOVE_COL_TARGET))) {
             this._layoutBuilder.cancelMove();
             this._detachCancelMoveFieldEvents();
+            this._enableAddFields();
```

Comment:
Won't the `cancelMove` call already cause `_enableAddFields` to be called through `_afterLayoutBuilderMoveEnd`?